docs/UserGuide.md

```diff
@@ -136,28 +162,28 @@ An example:
 
 1) Create a workload with gold policy.
 
-```
-curl -H "Content-Type: application/json" --request POST --data \
+```shell
+$ curl -H "Content-Type: application/json" --request POST --data \
          '{"task_ids":["78377"], "policy": "gold"}' \
```

Comment:
Hmm.. yep, we can refine this better.

A user may try the command line without thinking (at least for me, I will do as what the user guide tell me), so better to tell all steps when we ask the user to try the first command ::

1) Create a workload with gold policy, say you have a task running (task id is 78377).